src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -54,19 +56,20 @@ object ProjectBuilder {
 
     val dockerfiles = templefile.servicesWithPorts.map {
       case (name, service, port) =>
-        val dockerfileRoot     = DockerfileBuilder.createServiceDockerfile(name.toLowerCase, service, port)
+        val dockerfileRoot     = DockerfileBuilder.createServiceDockerfile(name.toLowerCase, service, port.service)
         val dockerfileContents = DockerfileGenerator.generate(dockerfileRoot)
         (File(s"${name.toLowerCase}", "Dockerfile"), dockerfileContents)
     }
 
     val orchestrationRoot = OrchestrationBuilder.createServiceOrchestrationRoot(
       StringUtils.kebabCase(templefile.projectName),
-      templefile.servicesWithPorts.toSeq,
+      templefile.servicesWithPorts.map { case (name, block, ports) => (name, block, ports.service) }.toSeq,
```

Comment:
Not a fan of this, but not sure how it could be made nicer. Maybe wrap it up into `templefile.servicesWithServicePorts`?

Reply:
This was somewhat intentional - Kube will eventually need to know about the metrics ports too so it should be able to take the Ports type instead of Int. Could do that in a follow up?

Reply:
ok, fair enough. I wonder if it’s worth making a case class for the `(name, block, ports)` triple